### DISCUSSION TITLE
Measure tool >?

### DISCUSSION THREAD

**Discussion by KennyBeaudoin** [supermerill/SuperSlicer#3331] [Q&A]
Hello .. id like to know if you could include a measure tool in the slicer as there is one in cura ... this is very usefull when we just want to resize a part and measure it up or down or change some settings but not all the same axis at same time .. thank you

**Comment by neophyl** [comment]
https://github.com/supermerill/SuperSlicer/issues/1625

**Reply by KennyBeaudoin** [reply]
its not what im looking for and this is still not done since 2021 ? i dont understand why the team did not put a bit more work in this tool as a lot of us were using it in cura :(

**Reply by neophyl** [reply]
The team ?  You mean the single developer ?

please have a read of https://github.com/prusa3d/PrusaSlicer/issues/3651 where this feature has also been requested and you will see just how much history it has behind it. 

Of course you could code it up yourself and submit a pull request if it’s taking too long for you.

**Comment by Visthum** [comment]
> Hello .. id like to know if you could include a measure tool in the slicer as there is one in cura ... this is very usefull when we just want to resize a part and measure it up or down or change some settings but not all the same axis at same time .. thank you

I'm using another program to measure the model, its very light weight and a lot better than the cura measurement tool.
And in SS I'm using a bed texture with cm grid to give me an estimate

**Reply by Visthum** [reply]
3D Tools is what I am using btw. It's free, super fast and also allows you to scale the object and all the measuring points will be updated. Tbh measuring inside a slicer is just a waste of time and energy.
